docs/modules/operation/pages/service-assurance/monitors/PageSequenceMonitor.adoc

```diff
@@ -53,42 +53,42 @@ Additionally, it is possible to configure a page sequence containing a path thro
 
 This monitor implements the <<service-assurance/monitors/introduction.adoc#ga-service-assurance-monitors-common-parameters, Common Configuration Parameters>>.
 
-.Page Sequence Parameter
+.Page Parameter
 [options="header, autowidth"]
 |===
 | Parameter                  | Description                                                                                | Required | Default
-| `name`                     | The name of the page-sequence. *(Is this relevant/used?)*                                  | optional | `-`
-| `method`                   | _HTTP_ method for example _GET_ or _POST_                                                  | `-`      | `-`
+| `name`                     | The name of the page-sequence.                                   | optional | `-`
```

Comment:
```suggestion
| `name`                     | The name of the page sequence.                                   | optional | `-`
```

Reply:
Unless this is modifying something?

Reply:
This isn't really modifying anything - just removing the "is this relevant" comment.